content/tutorials/jwt-authentication.textile

```diff
@@ -136,135 +144,150 @@ This step shows you how to create the server code from scratch. If you get stuck
 npm install jsonwebtoken express
 ```
 
-3. Copy your API key to the clipboard, from your Ably Dashboard. It will be of the form @12AB3a.ab3GOm:c-B4Emryq9aFmOxP@.
+3. Copy your API key to the clipboard, from your Ably Dashboard.
+
+It will be of the form @<app ID>.<key ID>:<key secret>@.
+For example: @12AB3a.ab3GOm:c-B4Emryq9aFmOxP@.
 
 4. Create a @.env@ file with the following content:
 
-```
-API_KEY=your_api_key
+```[sh]
+ABLY_API_KEY=your_api_key
 ```
 
-5. Paste in your API key to replace @your_api_key@.
+Paste in your API key to replace @your_api_key@.
 
-6. Create a new file called @server.js@. Add the following code at the top of the file:
+5. Create a new file called @server.js@. Add the following code at the top of the file:
 
 ```[javascript]
-const jwt = require("jsonwebtoken") 
-require('dotenv').config()
+const jwt = require("jsonwebtoken");
+require('dotenv').config();
 ```
 
-This includes the JWT library you'll be using, and also loads the environment file containing your @API_KEY@.
+This includes the JWT library you'll be using, and also loads the environment file containing your @ABLY_API_KEY@.
 
-7. Add the following code:
+6. Add the following code:
```

Comment:
At the bottom of step 6 there is an "add the following code:" that doesn't show any code 😕

Reply:
:scissors: